### PR TITLE
Skipping USDT tests for Power processor 

### PR DESCRIPTION
**Pull request by dubeyabhishek**
The support got added in 1f0f7b6, but later removed.

### CODE REVIEW

**Comment by chenhengqi** [comment]
How does it work ? It seems that this change only affects the test codes.

**Comment by dubeyabhishek** [comment]
Yes, this patch is only covering test code.

File: tests/python/include/folly/tracing/StaticTracepoint-ELF.h [hunk 1]
```diff
@@ -21,8 +21,12 @@
 
 // Default constraint for the probe arguments as operands.
 #ifndef FOLLY_SDT_ARG_CONSTRAINT
+#if defined(__powerpc64__) || defined(__powerpc__)
```
[the remark this answers is not in the document]
Comment (dubeyabhishek) [reply tests/python/include/folly/tracing/StaticTracepoint-ELF.h:24]: From the discussion at 1f0f7b6. the changes look sufficient. let me check, if we can have corresponding in with folly. Motivation of patch is to have the tests passing on power system.

[the remark this answers is not in the document]
Comment (dubeyabhishek) [reply tests/python/include/folly/tracing/StaticTracepoint-ELF.h]: For now, I am skipping the usdt tests for power systems to stabilize test suite.

**Comment by dubeyabhishek** [comment]
@chenhengqi can we close this PR?

**Comment by dubeyabhishek** [comment]
@yonghong-song can we close this PR? Please let me know if anything else is needed.

**Comment by yonghong-song** [comment]
The current usdt tests rely on `folly/tracing/StaticTracepoint.h` which only supports x86. All other architectures will fail these usdt tests. I am okay with this patch. But in the future hopefully other arch can modify folly/tracing files to add other arch support so usdt can be tested in their respective arch'es.

**Pull request merged**